src/backend/controllers/book_test.go

```diff
@@ -12,61 +19,46 @@ func AssertExpected(t *testing.T, expected, received interface{}) {
 	}
 }
 
-func TestGetBooks(t *testing.T) {
-	expected := GenerateSampleBooks()
-	received := GetBooks()
+func SetupContext(db *gorm.DB) (*httptest.ResponseRecorder, *gin.Context) {
```

Comment:
Let's upgrade to use such as a beforeAll or after all functionality too this to make set-up and tear-down easier.

Reply:
Turns out, as of Golang 1.14, there is no setup / beforeAll style functionality built  into the testing suite. Without looking to external tools, I think it's best that we simply resolve and leave the function setup call as is.